services/fileServices/MdPageServices/__tests__/UnlinkedPageService.spec.js

```diff
@@ -0,0 +1,168 @@
+describe("Unlinked Page Service", () => {
+  const siteName = "test-site"
+  const accessToken = "test-token"
+  const fileName = "test-file"
+  const directoryName = "pages"
+  const mockContent = "test"
+  const mockMarkdownContent = "---test---"
+  const mockFrontMatter = {
+    title: "fileTitle",
+    permalink: "file/permalink",
+  }
+  const sha = "12345"
+
+  const reqDetails = { siteName, accessToken }
+
+  const mockGithubService = {
+    create: jest.fn(),
+    read: jest.fn(),
+    update: jest.fn(),
+    delete: jest.fn(),
+  }
+
+  jest.mock("@utils/markdown-utils", () => ({
+    retrieveDataFromMarkdown: jest.fn().mockReturnValue({
+      frontMatter: mockFrontMatter,
+      pageContent: mockContent,
+    }),
+    convertDataToMarkdown: jest.fn().mockReturnValue(mockMarkdownContent),
+  }))
+  const {
+    UnlinkedPageService,
+  } = require("@services/fileServices/MdPageServices/UnlinkedPageService")
+  const service = new UnlinkedPageService({
+    gitHubService: mockGithubService,
+  })
+  const {
+    retrieveDataFromMarkdown,
+    convertDataToMarkdown,
+  } = require("@utils/markdown-utils")
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  describe("Create", () => {
+    mockGithubService.create.mockReturnValue({ sha })
```

Comment:
We should use mockResolvedValue for mocked async calls

Reply:
Resolved in abe1948